src/billing/components/BillingPage.tsx

```diff
@@ -8,20 +8,28 @@ import RateLimitAlert from 'src/cloud/components/RateLimitAlert'
 import LimitChecker from 'src/cloud/components/LimitChecker'
 import BillingProvider from 'src/billing/context/billing'
 
-const BillingPage: FC = () => (
-  <BillingProvider>
-    <Page titleTag="Billing">
-      <Page.Header fullWidth={false} testID="billing-page--header">
-        <Page.Title title="Billing" />
-        <LimitChecker>
-          <RateLimitAlert />
-        </LimitChecker>
-      </Page.Header>
-      <Page.Contents scrollable={true} testID="billing-page-contents--scroll">
-        <BillingPageContents />
-      </Page.Contents>
-    </Page>
-  </BillingProvider>
-)
+// Utils
+import {event} from 'src/cloud/utils/reporting'
+
+const BillingPage: FC = () => {
+  useEffect(() => {
+    event('billing_page.viewed')
+  }, [])
```

Comment:
I'm a little hesitant to include eventing in the BillingPage for the sake of adding eventing. I understand that it was part of the original issue's criteria, but considering the fact that it's a feature flagged WIP, it seems a bit random to include an event on a page that no one is hitting. In addition, I'm not certain that providing an event for viewing the billing page will provide any specific insight on how the user is interacting with the page. I'd rather leave event for the UI Unification work scoped within that epic rather than include disparate events

Reply:
If an event in BillingPage is a discussible topic, I will skip it for now.